src/modules/m_permchannels.cpp

```diff
@@ -221,6 +221,12 @@ class ModulePermanentChannels : public Module
 
 				ServerInstance->Logs->Log(MODNAME, LOG_DEBUG, "Added %s with topic %s", channel.c_str(), c->topic.c_str());
 
+				if (modes.find('P') == std::string::npos)
+				{
+					ServerInstance->Logs->Log("m_permchannels", LOG_DEFAULT, "%s (%s) does not have +P set in <permchannels:modes>; it will be deleted when empty!",
+						c->name.c_str(), tag->getTagLocation().c_str());
+				}
+
```

Comment:
Yeah, theres a PR open for that already. When this gets merged i'll remove the notice in the other PR.